app/src/main/res/xml/changelog_master.xml

```diff
@@ -1,15 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
 <changelog xmlns:tools="http://schemas.android.com/tools"
     tools:ignore="MissingDefaultResource">
-    <release version="2025.6.3 - Main" versioncode="3">
+    <release version="2025.6.4 - Main" versioncode="3">
         <change>The app now uses all available space on the screen (edge-to-edge support)</change>
         <change>Viewing more info for entities from device controls, notifications, shortcuts and tiles is now more responsive</change>
+        <change>Support invitation links from my.home-assistant.io</change>
+        <change>Redirect links from my.home-assistant.io directly open the app without confirmation</change>
         <change>Bug fixes and dependency updates</change>
     </release>
-    <release version="2025.6.3 - Wear" versioncode="2">
+    <release version="2025.6.4 - Wear" versioncode="2">
         <change>Bug fixes and dependency updates</change>
     </release>
-    <release version="2025.6.3 - Automotive" versioncode="1">
+    <release version="2025.6.4 - Automotive" versioncode="1">
         <change>Bug fixes and dependency updates</change>
```

Comment:
This feature should also work for Automotive right? Might be nice to mention it so they know they dont have to type in the full URL and can use an invite link too?

Reply:
Good point 👍